node/src/reactor.rs

```diff
@@ -406,8 +408,8 @@ where
                 .await
                 .expect("could not serialize snapshot");
 
-            let mut file =
-                File::create("queue_dump_debug.txt").expect("could not create dump file");
```

Comment:
If queue dumps are large, they might fill up `/tmp`, which may be limited to 1-2 GB in size on many distros using tmpfs.

Should we maybe try an `or_else` here? We try to dump to the local file first, if that fails try dumping to `/tmp`, if that fails panic?

We could also just log our failure to dump as an error, allowing the user to clear up space/whatever and send the SIGUSR1 again.